backends/core/models.py

```diff
@@ -226,13 +287,19 @@ class Job(BaseModel):
       primaryjoin='Job.id==StartCondition.preceding_job_id',
       secondaryjoin='StartCondition.job_id==Job.id')
   enqueued_workers_count = Column(Integer, default=0)
-  succeeded_workers_count = Column(Integer, default=0)
-  failed_workers_count = Column(Integer, default=0)
 
   def __init__(self, name=None, worker_class=None, pipeline_id=None):
+    self.memcache_client = memcache.Client()
     self.name = name
     self.worker_class = worker_class
     self.pipeline_id = pipeline_id
+  
+  @orm.reconstructor
+  def init_on_load(self):
+    self.memcache_client = memcache.Client()
+  
+  def _build_pipeline_prefix(self):
```

Comment:
Probably needs to be renamed `_build_job_prefix`?
And the logic needs to be updated to include the pipeline id and job id I guess.

Reply:
should have been self.pipeline_id indeed. Interesting the tests have not been affected though. Must have been the same id. probably should be fixed